### PR TITLE
update prod-host-setup.md for AMD

### PR DESCRIPTION
**Pull request by serban300**
## Reason for This PR

update prod-host-setup.md for AMD

## Description of Changes

update prod-host-setup.md for AMD

- [ ] This functionality can be added in [`rust-vmm`](https://github.com/rust-vmm).

## License Acceptance

By submitting this pull request, I confirm that my contribution is made under
the terms of the Apache 2.0 license.

## PR Checklist

`[Author TODO: Meet these criteria.]`
`[Reviewer TODO: Verify that these criteria are met. Request changes if not]`

- [x] All commits in this PR are signed (`git commit -s`).
- [x] The reason for this PR is clearly provided (issue no. or explanation).
- [x] The description of changes is clear and encompassing.
- [x] Any required documentation changes (code and docs) are included in this PR.
- [x] Any newly added `unsafe` code is properly documented.
- [x] Any API changes are reflected in `firecracker/swagger.yaml`.
- [x] Any user-facing changes are mentioned in `CHANGELOG.md`.
- [x] All added/changed functionality is tested.

### CODE REVIEW

File: docs/prod-host-setup.md [hunk 1]
```diff
@@ -64,7 +64,7 @@ Verification can be done by running:
 (grep -q "^forceoff$\|^notsupported$" /sys/devices/system/cpu/smt/control && echo "Hyperthreading: DISABLED (OK)") || echo "Hyperthreading: ENABLED (Recommendation: DISABLED)"
 ```
 
-#### Check Kernel Page-Table Isolation (KPTI) support
```
Comment (raduiliescu) [inline docs/prod-host-setup.md:67]: Not sure if this is the best way to put it.
I would say [Intel only] and maybe add a phrase in the begining that platform specific info will be put in brackets...

Comment (serban300) [reply docs/prod-host-setup.md:67]: Done.

File: docs/prod-host-setup.md [hunk 2]
```diff
@@ -47,6 +47,9 @@ When deploying Firecracker microVMs to handle multi-tenant workloads, the
 following host environment configurations are strongly recommended to guard
 against side-channel security issues.
 
+Some of the mitigations are platform specific. When appliable, this information
```
Comment (georgepisaltu) [inline docs/prod-host-setup.md:50]:
```suggestion
Some of the mitigations are platform specific. When applicable, this information 
```

Comment (serban300) [reply docs/prod-host-setup.md]: Fixed.

**Review by raduiliescu — APPROVED** [review]

**Review by georgepisaltu — APPROVED** [review]

**Pull request merged**